core/src/main/java/org/jboss/pnc/causeway/brewclient/BuildTranslatorImpl.java

```diff
@@ -48,14 +46,18 @@
 
     @Override
     public KojiImport translate(BrewNVR nvr, BuildRecordRest build, BuildArtifacts artifacts) throws CausewayException {
+        String[] splittedName = nvr.getName().split("-");
```

Comment:
maven allows group ids with dashes, so split("-", 2) also won't solve the problem.